src/auth0-sso-login.js

```diff
@@ -23,11 +23,10 @@ export default class auth {
 
   /**
    * @description update the detailed profile with a call to the Auth0 Management API
-   * @param auth0AccessToken The client specific audience token to call Auth0 Management
    * @return {Promise<any>} resolved promise with user profile; rejected promise with error
    */
-  refreshProfile(auth0AccessToken) {
-    return this.getProfile(auth0AccessToken)
+  refreshProfile() {
```

Comment:
There's a call in `renewAuth` to `refreshProfile` which passes in the token. While your change will still work, it'd be cleaner to remove it there. The rest looks good.